README.md

```diff
@@ -67,23 +67,39 @@ $ $GOPATH/bin/terraform-provider-ionoscloud
 ...
 ```
 
+## Testing the Provider
+
+⚠️ **Warning:** Acceptance tests create real resources, and often cost money to run.
```

Comment:
In terraform, there are only acceptance tests? I had the impression we have e2e tests.

Reply:
In this case acceptance tests involves e2e tests, since they create real resources